fido-device-onboard.spec

```diff
@@ -70,7 +70,11 @@ install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-admin-to
 install -D -m 0755 -t %{buildroot}%{_bindir} target/release/fdo-owner-tool
 install -D -m 0755 -t %{buildroot}%{_bindir} target/release/fdo-admin-tool
 install -D -m 0644 -t %{buildroot}%{_unitdir} examples/systemd/*
-install -D -m 0644 -t %{buildroot}%{_docdir}/fdo examples/config/*
+# we do not need the rendezvous-info.yml for the AIO command, add everything else
+install -D -m 0644 -t %{buildroot}%{_docdir}/examples examples/config/manufacturing-server.yml
```

Comment:
you now have two examples and still a space :) so basically "sed -i 's/examples //' fido-device-onboard.spec"

Reply:
The only space that I can see is the space that separates the <dest-dir> from the <source-dir>, what other space is there that I can't see?

Reply:
Yep, you're right, I was reading it wrong :-/

Reply:
No problem!